Source/Library/Adapt.Analytics/AngleConversionAnalytic.cs

```diff
@@ -0,0 +1,108 @@
+//  file except in compliance with the License. You may obtain a copy of the License at:
+//
+//      http://opensource.org/licenses/MIT
+//
+//  Unless agreed to in writing, the subject software distributed under the License is distributed on an
+//  "AS-IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. Refer to the
+//  License for the specific language governing permissions and limitations.
+//
+//  Code Modification History:
+//  ----------------------------------------------------------------------------------------------------
+//  08/02/2021 - C. Lackner
+//       Generated original version of source code.
+//  12/10/2021 - A. Hagemeyer
+//       Changed to an angle conversion analytic
+//
+// ******************************************************************************************************
+
+
+using Adapt.Models;
+using GemstoneCommon;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Numerics;
+using System.Runtime;
+using System.Threading.Tasks;
+using System.Windows;
+
+namespace Adapt.DataSources
+{
+    /// <summary>
+    /// Convert between radians and degrees
+    /// </summary>
+
+    [AnalyticSection(AnalyticSection.DataCleanup)]
+
+    [Description("Angle Conversion: Convert between radians and degrees")]
+    public class AngleConversion : IAnalytic
+    {
+        private Setting m_settings;
+        private int m_fps;
+
+        public class Setting
+        {
+            public AngleUnit Unit { get; set; }
+        }
+
+        public Type SettingType => typeof(Setting);
+
+        public int FramesPerSecond => m_fps;
+
+        int IAnalytic.PrevFrames => 0;
+
+        int IAnalytic.FutureFrames => 0;
+
+        public IEnumerable<string> OutputNames()
+        {
+            return new List<string>() { "Converted" };
+        }
+
+        public IEnumerable<string> InputNames()
+        {
+            return new List<string>() { "Original" };
+        }
+
+        public Task<ITimeSeriesValue[]> Run(IFrame frame, IFrame[] previousFrames, IFrame[] futureFrames)
+        {
+            AdaptValue result = new AdaptValue("Converted", getAngleConversion(frame), frame.Timestamp);
+            return Task.FromResult<ITimeSeriesValue[]>(new AdaptValue[] { result });
+        }
+
+        public double getAngleConversion(IFrame frame) 
+        {
+            ITimeSeriesValue original = frame.Measurements["Original"];
+            if (m_settings.Unit == AngleUnit.Degrees)
+                return (Math.PI / 180) * original.Value;
+            else
+                return (180 / Math.PI) * original.Value;
+        }
+
+        public void Configure(IConfiguration config)
+        {
+            m_settings = new Setting();
+            config.Bind(m_settings);
+        }
+
+        public void SetInputFPS(IEnumerable<int> inputFramesPerSecond) 
```

Comment:
Not neccesarry. Single Input Analytics can just return `inputFramesPerSecond.First()`